ebos/eclmpiserializer.hh

```diff
@@ -194,6 +197,45 @@ public:
 
     }
 
+    //! \brief Handler for std::optional.
+    //! \tparam T Type for data
+    //! \param data The optional to (de-)serialize
+    template<class T>
+    void optional(const std::optional<T>& data)
+    {
+        if (m_op == Operation::PACKSIZE) {
```

Comment:
There is support for similar operations for *non-complex* types already in Parallellrestart.cpp - will that become a conflict?

Reply:
it will not be in conflict. they are simply not reachable now, so the optional support can in principle be removed from there now.